lib/funiq/Matcher.h

```diff
@@ -60,12 +61,17 @@ void Matcher::add(std::string line) {
 	}
 }
 
+// todo: count max width necessary
 void Matcher::show(std::ostream* output) {
 	for(auto matchPair : *matchMap) {
 		StringList v = *matchPair.second;
 		bool first = true;
 		for(std::string matchItem : v) {
 			if(first || _settings.showAllMatches) {
+				if(first && _settings.showTotals)
+					*output << 
+						std::setw(_settings.totalsFieldWidth) <<
```

Comment:
Very neatly done. I was somehow unaware of `std::setw` up until this point as well, so you've taught me something. Thanks!